dataframely/collection.py

```diff
@@ -561,6 +561,42 @@ def filter(
 
         return cls._init(results), failures
 
+    def join(
+        self,
+        primary_keys: pl.LazyFrame,
+        how: Literal["semi", "anti"] = "semi",
```

Comment:
Should we also add the `maintain_order` parameter? 
```suggestion
        how: Literal["semi", "anti"] = "semi",
        maintain_order: Literal["none", "left"] = "none",
```

Reply:
Yes makes sense 👍